spec/features/multi_factor_authentication/mfa_cta_spec.rb

```diff
@@ -70,5 +71,34 @@
       click_link(t('mfa.second_method_warning.link'))
       expect(page).to have_current_path(authentication_methods_setup_path)
     end
+
+    it 'At setup, phone as second MFA show a cancel link that returns to mfa setup' do
+      allow(IdentityConfig.store).to receive(:platform_auth_set_up_enabled).and_return(true)
+      allow(IdentityConfig.store).
+        to receive(:show_unsupported_passkey_platform_authentication_setup).
+        and_return(true)
+
+      user = sign_up_and_set_password
+      user.password = Features::SessionHelper::VALID_PASSWORD
```

Comment:
`sign_up_and_set_password` does not need to be assigned to a user because it also creates a password

```suggestion
      sign_up_and_set_password
```